AUTHORS

```diff
@@ -1,11 +1,18 @@
 Abishek Raju <abishek.raju@gmail.com>
+Amin Mosayyebzadeh <mosayyeb@bu.edu>
 Andrew Mohn <AndrewMohn@gmail.com>
 George Silvis, III <george.iii.silvis@gmail.com>
+Ian Ballou ianballou67@gmail.com
 Ian Denhardt <ian@zenhack.net>
 Igibek Koishybayev <koishybayev.igibek@gmail.com>
 Jay Hennessey <henn@users.sourceforge.net>
+(Jethro) Shuwen Sun <http://orcid.org/0000-0001-6811-8310>
 Jonathan Bell <jbell@bu.edu>
+Kristi Nikolla <knikolla@massopen.cloud>
+Kyle Hogan <klhogan@bu.edu>
 Logan Bernard <bernard.logan4@gmail.com>
+Lucas Xu <xuh@massopen.cloud>
+Naved Ansari <naved001@massopen.cloud>
```

Comment:
I don't have that email address. you could put my gmail or bu email, it's naved001@gmail.com/bu.edu

Reply:
free like everybody is not using their regular email addresses here. You sure you want to use the gmail? Why not get a massopen.cloud email from Rado just saying ;)

Reply:
Not sure I follow the point about "everybody is not using their regular email addresses here". I only see three massopen.cloud addresses, and all of them are introduced by this patch.

Reply:
Sorry for the confusion. I meant many people are using the email addresses that they don't check on a regular basis. And IMHO, hardcoded email addresses is very bad since it is very easy to be scraped and receive spams.